### PR TITLE
Mongo filter

### PR DESCRIPTION
**Pull request by puinenveturi**

### CODE REVIEW

File: src/proxy/mongoCore.js [hunk 1]
```diff
@@ -21,7 +23,15 @@ module.exports = function(dependencies) {
 			throw new Error("config.model is mandatory!");
 		}
 
-		var Model = config.model;
+		let Model = config.model;
+		let populate = config.populate || null;
+		let populateFields = String();
```
Comment (gyulanemeth) [inline src/proxy/mongoCore.js]: = "" would be nicer.

**Comment by puinenveturi** [comment]
@gyulanemeth pls merge this

**Comment by puinenveturi** [comment]
have merged master into this branch

File: spec/proxy/mongoSpec.js [hunk 2]
```diff
@@ -32,40 +64,12 @@ describe("Mongo proxy", function() {
 	});
 
 	describe("with valid config", function() {
-		var mongoProxy;
+		let mongoProxy;
 
 		beforeAll(function(done) {
 			spyOn(mockAsync, "parallel").and.callThrough();
 
-			var mockModel = {
```
Comment (gyulanemeth) [inline spec/proxy/mongoSpec.js:40]: Why did you put it to farther on the closure?

**Pull request closed without merging**